results_paper/results.ipynb

```diff
@@ -0,0 +1,1237 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# wfd = pd.read_csv('WFDtruth.csv')\n",
+    "# wfd.set_index('object_id', inplace=True)\n",
+    "# ddf = pd.read_csv('DDFtruth.csv')\n",
+    "# ddf.set_index('object_id', inplace=True)\n",
+    "# in_val = wfd.append(ddf)\n",
+    "# in_val.to_csv('submissions/validation/truth.csv', header=True)\n",
+    "\n",
+    "# wfd_probs = pd.read_csv('submissions/validation/WFDprobs.csv').set_index('object_id')\n",
+    "# ddf_probs = pd.read_csv('submissions/validation/DDFprobs.csv').set_index('object_id')\n",
+    "# val_probs = wfd_probs.append(ddf_probs)\n",
+    "val_probs = pd.read_csv('0_validation.csv').set_index('object_id')\n",
+    "# print(val_probs.isnull().values.sum())\n",
+    "# print(np.any(np.isnan(val_probs)))\n",
+    "val_probs.to_csv('submissions/validation/validation_probs.csv', header=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # # Run me only if you've never run the notebook before. (slow)\n",
+    "# for i in label_dict.keys():\n",
+    "#     one_class = val_probs.loc[in_val[in_val['true_target'] == i].index, :]\n",
+    "#     if ((np.all(one_class == epsilon) or np.any(np.isnan(one_class)))):\n",
+    "#         print('arrr, thar be NaNs or all 0s!')\n",
+    "#         one_class.fillna(epsilon)\n",
+    "#         one_class.replace(to_replace=0, value=epsilon)\n",
+    "#         one_class.replace(to_replace=0., value=epsilon)\n",
+    "# #     assert ~((np.all(one_class == epsilon) or np.any(np.isnan(one_class))))\n",
+    "#     if ~((np.all(one_class == epsilon) or np.any(np.isnan(one_class)))):\n",
+    "#         normalized = one_class.sum(axis=1)\n",
+    "#     else:\n",
+    "#         print('normalization failed for '+str(i)+' with all '+str(len(one_class))+' zeros='+str(np.all(one_class == epsilon))+' and NaNs='+str(np.any(np.isnan(one_class))))\n",
+    "#         normalized = 1.\n",
+    "#     to_save = one_class.divide(normalized, axis=0)\n",
+    "#     pred_class_inds = {i: int(i[6:]) for i in to_save.columns.values}\n",
+    "#     to_save.rename(columns=pred_class_inds, inplace=True)\n",
+    "#     to_save.to_csv(os.path.join('submissions/validation', 'probvecs'+str(i)+'true.csv'), header=True)\n",
+    "# #     print('saved validation class '+str(i))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Classification probabilities\n",
+    "\n",
+    "This is just for the winning submission for now.\n",
+    "You have to get the files with the submission results first and put them in the same directory as this notebook.\n",
+    "_(I don't know if it's okay to post the link to where those live or if it's private and will be published with the paper through Zenodo or something.)_"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
There were some issues for me on the format of the csv for the different contestants, and I used the reformatted CSV files from @mi-dai  - are you using them here? If so we should include a note